.github/workflows/testing.yml

```diff
@@ -62,8 +63,8 @@ jobs:
           mkdir -p ~/.abinit/pseudos
           cp -r tests/test_data/abinit/pseudos/ONCVPSP-PBE-SR-PDv0.4 ~/.abinit/pseudos
           pip install .[strict,tests,abinit]
-          pip install torch-runstats
           pip install --no-deps nequip==0.5.6
+          pip install tblite==0.3.0
```

Comment:
should `tblite` be added to pyproject

Reply:
Yeah, I'll put this in an `ase-ext` requirements. The base ASE stuff does not require `tblite` in any way